extended/src/test/java/apoc/ml/OpenAILocalAIIT.java

```diff
@@ -70,4 +139,16 @@ private Map<String, Object> getParams(String model) {
                         MODEL_CONF_KEY, model)
         );
     }
+
+    private Map<String, Object> getParams(String model, String schema, String question) {
+        return Util.map(
+                "apiKey", "x",
+                "conf", Map.of(
+                        ENDPOINT_CONF_KEY, localAIUrl,
+                        MODEL_CONF_KEY, model
+                ),
+                "schema", schema,
+                "question", question
+        );
```

Comment:
Prova a richiamare l'altro getParams(model), tipo in questo modo:

```suggestion
        var params = Util.map(
                "schema", schema,
                "question", question
        );
        params.putAll( getParams(model) );
        return params;
```

Reply:
Così non va in errore perché vorrebbe che getParams restituisca Map<String, String>

Reply:
Prova a vedere se c'è un altro modo, tipo mettendo Map<String, Object> al posto di var, oppure ritornando Map<String, String>